mlog/log.go

```diff
@@ -51,6 +52,20 @@ type Logger struct {
 	fileLevel    zap.AtomicLevel
 }
 
+// LogWriter is an io.Writer that writes through t.Log
+type LogWriter struct {
+	tb testing.TB
+}
+
+func (lw *LogWriter) Write(b []byte) (int, error) {
+	n := len(b)
+	s := string(b)
+	lw.tb.Log(s)
+
+	// Assuming that the logs are written succesfuly
```

Comment:
I think this is a perfectly fine assumption to make in this context, but a typo:
```suggestion
	// Assuming that the logs are written successfully
```